docs/pages/submit/ios.mdx

```diff
@@ -1,99 +1,213 @@
+  />
+</Collapsible>
 
-<Step label="1">
-## Build a standalone app
+Learn about all the options you can provide in the [eas.json reference](/eas/json/#ios-specific-options-1).
 
-You'll need a native app binary signed for store submission. You can either use [EAS Build](/build/introduction/) or do it on your own.
+To speed up the submission process, you can use the `--auto-submit` flag to automatically submit a build after it's built:
 
-</Step>
+<Terminal cmd={['$ eas build --platform ios --auto-submit']} />
 
-<Step label="2">
+Learn more about the `--auto-submit` flag in the [automate submissions](/build/automate-submissions/) guide.
 
-## Start the submission
+## Submitting your app using CI/CD services
 
-To submit the binary to the App Store, run `eas submit -p ios` from inside your project directory. The command will lead you step by step through the process of submitting the app. See the [Configuration with eas.json](/build/eas-json/) page to learn how to pre-configure your submission.
+<Prerequisites numberOfRequirements={5}>
+  <Requirement number={1} title="Sign up for an Apple Developer account">
+    An Apple Developer account is required to submit your app to the Apple App Store. You can sign up for an Apple Developer account on the [Apple Developer Portal](https://developer.apple.com/account/).
+  </Requirement>
+  <Requirement number={2} title="Include a bundle identifier in app.json">
+    Include your app's bundle identifier in **app.json**:
 
-> Although it's possible to upload any binary to the store, each submission is associated with an Expo project. That's why it's important to start a submission from inside your project's directory because [app config](/workflow/configuration) is defined inside that directory.
+    ```json app.json
+    {
+      "ios": {
+        "bundleIdentifier": "com.yourcompany.yourapp"
+      }
+    }
+    ```
 
-If you have not generated an App Store Connect API Key yet, you can let EAS CLI take care of that for you by signing into your Apple Developer Program account and following the prompts. You can also upload your own [API Key](https://expo.fyi/creating-asc-api-key) or pass in an [Apple app-specific password](https://expo.fyi/apple-app-specific-password).
+  </Requirement>
+  <Requirement number={3} title="Configure your App Store Connect API Key">
+    Run the following command to configure your App Store Connect API Key:
 
-To upload your iOS app to the Apple App Store, run `eas submit --platform ios` and follow the instructions on the screen.
+    <Terminal cmd={['$ eas credentials --platform ios']} />
 
-The command will perform the following steps:
+    The command will prompt you to select the type of credentials you want to configure.
+    1. Select the `production` build profile
+    2. Log in with your Apple Developer account and follow the prompts
+    3. Select "App Store Connect: Manage your API Key"
+    4. Select "Set up your project to use an API Key for EAS Submit"
```

Comment:
```suggestion
    3. Select **App Store Connect: Manage your API Key**
    4. Select **Set up your project to use an API Key for EAS Submit**
```

Consistency nit: Let's make them bold since they are actions to be performed by the user when running the command. This is a pattern we follow for these kind of instructions.